onnx/defs/parser.cc

```diff
@@ -7,6 +7,7 @@
 
 #include "onnx/defs/parser.h"
 
+#include <cctype>
```

Comment:
I wonder why it was working before.

Reply:
Maybe it's OK in newer versions. I'm using MSVC2017

Reply:
CI works so I guess it is working with newer versions. It would be safer if you add one comment why it was added. A linter could detect it is not needed in the future and the line could be removed.

Reply:
I guess it is also a relatively new change that was introduced just a month or two ago. Seems useful to add all relevant includes. I thought there was a specific CI introduced for such missing includes ....

Reply:
I don't think feature from https://github.com/onnx/onnx/pull/5528 will be removed easily so I feel this header should be treated like other file: https://github.com/onnx/onnx/blob/c85deb736aff79e005c0695c859e47a33ff6d514/onnx/defs/data_type_utils.cc#L7